src/ssz/type/container.zig

```diff
@@ -501,6 +515,20 @@ pub fn VariableContainerType(comptime ST: type) type {
             }
         };
 
+        pub fn serializeIntoJson(allocator: std.mem.Allocator, writer: anytype, in: *const Type) !void {
+            try writer.beginObject();
+            inline for (fields) |field| {
+                const field_value = @field(in, field.name);
```

Comment:
```suggestion
                const field_value_ptr = &@field(in, field.name);
```

thi avoids an unnecessary copy, see #35